src/NServiceBus.AcceptanceTests/Core/PublishSubscribe/When_disabling_publishing.cs

```diff
@@ -52,9 +52,6 @@ public EndpointWithDisabledPublishing()
             {
                 var template = new DefaultServer();
                 template.TransportConfiguration = new ConfigureEndpointAcceptanceTestingTransport(false, true);
-                // use a persistence which doesn't support subscription persistence
-                template.PersistenceConfiguration = new ConfigureEndpointLearningPersistence(false);
```

Comment:
I've cleaned up the changes further and it seems like we don't need to use any persistence in this test?

Reply:
Strike that, I was confused :)